qiskit/_quantumjob.py

```diff
@@ -82,12 +80,10 @@ def __init__(self, circuits, backend=None,
             # circuits is actually a qobj...validate (not ideal but convenient)
             self.qobj = circuits
         else:
-            if not backend:
-                raise QISKitError('backend needs to be specified if '
-                                  'preformatted==True.')
             self.qobj = self._create_qobj(circuits, circuit_config, backend,
                                           seed, resources, shots, do_compile)
-        self.backend = self.qobj['config']['backend']
+        self.backend_name = self.qobj['config']['backend_name']
```

Comment:
`self.backend_name` is  not being used elsewhere, right?

Reply:
yep. removed.